apps/issuer/api.py

```diff
@@ -99,6 +99,59 @@ def put(self, request, **kwargs):
     def delete(self, request, **kwargs):
         return super(IssuerDetail, self).delete(request, **kwargs)
 
+class CollectionBadgeDetail(BaseEntityDetailView):
+    model = CollectionBadgeContainer
+    v1_serializer_class = CollectionBadgeClassSerializerV1
+    # v2_serializer_class = CollectionBadgeClassSerializerV2
+    permission_classes = (AuthenticatedWithVerifiedIdentifier, AuditedModelOwner)
+    valid_scopes = {
+        'get': ['r:issuer', 'rw:issuer'],
```

Comment:
Why do the scopes refer to issuers?

Reply:
I am not sure tbh, but the badgeclasses have the same scope and i tried to stay consistent with previously written code. Do you have another suggestion for the scopes?

Reply:
Hm weird, maybe at some point we need to give the scopes a general rewrite, but then it's probably fine for now